packages/kerosene-ui/package.json

```diff
@@ -40,17 +55,28 @@
     "jest-when": "^3.6.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-error-boundary": "^4.0.11",
     "rollup": "^4.5.1",
     "rollup-plugin-node-resolve": "^5.2.0"
   },
   "peerDependencies": {
+    "@tanstack/react-query": ">=5.0.0 <6.0.0",
     "react": ">=16.8.6",
-    "react-dom": ">=16.8.6"
+    "react-dom": ">=16.8.6",
+    "react-error-boundary": ">=3.0.0 <5.0.0"
+  },
+  "peerDependenciesMeta": {
+    "@tanstack/react-query": {
+      "optional": true
+    },
+    "react-error-boundary": {
```

Comment:
Didn't want to introduce the requirement for `@tanstack/react-query` and `react-error-boundary` onto all consumers of `@kablamo/kerosene-ui`, so have included these as optional peer dependencies rather than creating a whole separate package for boundaries. This does mean though that any consumers that do have these dependencies will need to have compatible versions.